upup/models/cloudup/resources/addons/networking.calico/v1.6.20161206.yaml

```diff
@@ -0,0 +1,307 @@
+# This ConfigMap is used to configure a self-hosted Calico installation.
```

Comment:
So the name of this is actually causing a bug.  Can we do v1.6 or v1.6.0?  I have to fix this with weave.